tests/test_schema.py

```diff
@@ -1404,6 +1404,30 @@ class UserModel(BaseModel):
     }
 
 
+def test_enum_str_default():
+    class MyEnum(str, Enum):
+        FOO = 'foo'
+
+    class UserModel(BaseModel):
+        friends: MyEnum = MyEnum.FOO
+
+    generated_schema_properties = UserModel.schema().get('properties', {})
+
+    assert generated_schema_properties.get('friends', {}).get('default', None) is MyEnum.FOO.value
```

Comment:
```suggestion
    assert UserModel.schema()['properties']['friends']['default'] is MyEnum.FOO.value
```

Reply:
No need for `get` method when you expect the key to be here

Reply:
Refactored.